pkg/identity/types.go

```diff
@@ -1,10 +1,62 @@
 // Package identity implements types and utility routines related to the identity of a workload, as used within OSM.
 package identity
 
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+const (
+	// namespaceNameSeparator used upon marshalling/unmarshalling MeshService to a string or vice versa
+	namespaceNameSeparator = "/"
+)
+
+var (
+	errInvalidMeshServiceFormat = errors.New("invalid namespaced service string format")
```

Comment:
Since there is no references to MeshService here, would you mind updating the error variable and the message to reflect this.

Reply:
Done! Thanks for catching this!